.github/workflows/get_layer_info_for_nf_imgs.yml

```diff
@@ -18,27 +18,78 @@ jobs:
 
       - name: Pull HTTP Response
         run: |
+          echo "Line $LINENO: starting the script"
           url_list=$(curl -s "https://raw.githubusercontent.com/uc-cdis/containers/master/nextflow-base-images/allowed_base_images.txt")
+          max_version_count=2  # Configurable value which indicates maximum of how many versions of past image hashes do we support
           TOKEN=$(curl -s https://public.ecr.aws/token/ | jq -r .token)
-          layer_json="{}"
+          output_file_name="nextflow-base-images/approved_img_layers_2.json" # change it once the xsoar script is ready to read the current file.
```

Comment:
Do not merge PR unless this is changed.

Reply:
also need to revert `# branches: master`